### PR TITLE
Add lib_gui_box

### PR DESCRIPTION
**Pull request by abenkovskii**
solves: #51

### CODE REVIEW

**Comment by abenkovskii** [comment]
I believe it's done.

File: library/lib_gui_box.ks [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+// This file is based on lib_window.ks from akrOS by akrasuski1.
+
+function draw_custom_gui_box {
+	parameter
+		x, y, w, h,
+		horizontal_char,
+		vertical_char,
+    corner_char.
+
+  // to-discuss: should we sanitize the input?
```
Comment (space-is-hard) [inline library/lib_gui_box.ks]: Yes, inputs should be bound to the parameters specified below; throw in a generic `HUDTEXT()` warning as well, such as "lib_gui_box exception: input out of range"

Comment (TDW89) [inline library/lib_gui_box.ks]: I will work on adding this into it later this evening (might end up as tomorrow)

**Comment by TDW89** [comment]
I am going to do a bit more work on this. I have bound the inputs but currently the box can be 1 high (a horizontal line) but not 1 wide (a vertical line), just going to change stuff around a little to get it able to create a vertical line.

**Pull request closed without merging**